src/components/students/students.scss

```diff
@@ -1,4 +1,21 @@
 .students {
-  background-color: lightgrey;
   width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: first baseline;
+  background-color: #eeeded;
+
+  &__title {
+    text-transform: uppercase;
+  }
+  &__searchBar {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+  }
+  &__studentCard {

```

Comment:
if the div of this class contains four cards it must never be name studentCard, maybe it should be named studentsCards or something similar